src/convertToObject.js

```diff
@@ -7,6 +7,20 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .trim()
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style)
```

Comment:
map and filter iteration are redundant.
it is possible to make everything in reduce method.